charts/victoria-logs-collector/_index.md.gotmpl

```diff
@@ -0,0 +1,124 @@
+---
+weight: 2
+title: VictoriaLogs Collector
+menu:
+  docs:
+    parent: helm
+    weight: 2
+    identifier: helm-victorialogs-collector
+url: /helm/victorialogs-collector
+aliases:
+  - /helm/victorialogs-collector/changelog/index.html
+tags:
+  - logs
+  - kubernetes
+---
+{{ template "chart.badges" . }}
+
+{{ template "chart.description" . }}
+
+{{ template "chart.prerequisites" . }}
+
+{{ include "chart.installSection" . }}
+
+## Chart Details
+
+This Helm chart deploys a log collection agent as a DaemonSet.
+It collects logs from all nodes in a Kubernetes cluster and forwards them to one or more VictoriaLogs instances.
```

Comment:
We should mention why it's not possible to override or change collector configuration.